index.xml

```diff
@@ -14044,66 +14181,126 @@ C implements A;</x:codeblock>
-                <li>Otherwise,
-                <ol>
-                <li>Initialize <var>values</var> to be an empty list of ECMAScript values.</li>
-                <li>Initialize <var>count</var> to 0.</li>
-                <li>Initialize <var>i</var> to 0.</li>
-                <li>While <var>i</var> &lt; <var>n</var>:
-                  <ol>
-                    <li>If <var>arg</var><sub><var>i</var></sub> is the special value “missing”, then append to <var>values</var> the ECMAScript <span class='estype'>undefined</span> value.</li>
-                    <li>Otherwise, <var>arg</var><sub><var>i</var></sub> is an IDL value.  Append to <var>values</var> the result of
-                      <a class='dfnref' href='#dfn-convert-idl-to-ecmascript-value'>converting</a> <var>arg</var><sub><var>i</var></sub> to an ECMAScript value,
-                      and set <var>count</var> to <var>i</var> + 1.</li>
-                    <li>Set <var>i</var> to <var>i</var> + 1.</li>
-                  </ol>
-                </li>
-                <li>Truncate <var>values</var> to have length <var>count</var>.</li>
-                <li>Let <var>script</var> be the <a class='dfnref' href='#dfn-callback-context'>callback context</a> associated with <var>V</var>.</li>
-                <li>Push <var>script</var> on to the <a class='dfnref external' href='http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html'>stack of incumbent scripts</a>. <a href='#ref-HTML'>[HTML]</a></li>
-                <li>Try running the following step:
-                  <ol>
-                    <li>Set <var>R</var> to the result of invoking the <span class='prop'>[[Call]]</span> method of <var>F</var>, providing the <a class='dfnref' href='#dfn-callback-this-value'>callback this value</a> as the <span class='esvalue'>this</span> value and <var>values</var> as the argument values.</li>
-                  </ol>
-                  And then, whether or not an exception was thrown:
-                  <ol>
-                    <li>Pop <var>script</var> off the <a class='dfnref external' href='http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html'>stack of incumbent scripts</a>.</li>
-                    <li>If an exception was thrown, end these steps, and allow it to propagate.</li>
-                  </ol>
-                </li>
-                </ol>
-                </li>
-                <li>If the callback function’s return type is <span class='idltype'>void</span>, return.</li>
-                <li>
-                  Return the result of <a class='dfnref' href='#dfn-convert-ecmascript-to-idl-value'>converting</a>
-                  <var>R</var> to an IDL value of the same type as the callback function’s return type.
-                </li>
+                <li>If the callback function's return type is <span class='idltype'>void</span>,
```

Comment:
This was here before, but it might use some explanation in a note...

The only way to end up with a non-callable here is to be dealing with a callback function which came from an attribute setter marked `[TreatNonObjectAsNull]`.  And in that case we just want to no-op the function call.